examples/step-60/doc/intro.dox

```diff
@@ -0,0 +1,253 @@
+finite dimensional system of equations:
+
+\f[
+\begin{pmatrix}
+K & C^T \\
+C & 0
+\end{pmatrix}
+\begin{pmatrix}
+u \\
+\lambda
+\end{pmatrix}
+=
+\begin{pmatrix}
+0 \\
+G
+\end{pmatrix}
+\f]
+
+where
+
+@f{eqnarray*}
+K_{ij} &:=& (\nabla v_j, \nabla v_i)_\Omega   & i,j=1,\dots,n \\
+C_{\alpha j} &:=& (v_j, q_\alpha)_\Gamma  &j=1,\dots,n, \alpha = 1,\dots, m \\\\
+G_{\alpha} &:=& (g, q_\alpha)_\Gamma & \alpha = 1,\dots, m.
+@f}
+
+While the matrix $K$ is the standard stiffness matrix for the Poisson problem on
+$\Omega$, and the vector $G$ is a standard right-hand-side vector for a finite
+element problem with forcing term $g$ on $\Gamma$, (see, for example, step-3),
+the matrix $C$ or its transpose $C^T$ are non-standard since they couple
+information on two non-matching grids.
+
+In particular, the integral that appears in the computation of a single entry of
+$C$, is computed on $\Gamma$. As usual in finite elements, we split this
+integral on each cell of the triangulation used to discretize $\Gamma$, we
+tranform the integral on $K$ to an integral on the reference element $\hat K$,
+where $F_{K}$ is the mapping from $\hat K$ to $K$, and compute the integral
+on $\hat K$ using a quadrature formula:
+
+\f[
+C_{\alpha j} := (v_j, q_\alpha)_\Gamma  = \sum_{K\in \Gamma} \int_{\hat K}
+\hat q_\alpha(\hat x) (v_j \circ F_{K}) (\hat x) J_K (\hat x) \mathrm{d} \hat x =
+\sum_{K\in \Gamma} \sum_{i=1}^{n_q}  \big(\hat q_\alpha(\hat x_i)  (v_j \circ F_{K}) (\hat x_i) J_K (\hat x_i) w_i \big)
+\f]
+
+Computing this sum is non-trivial because we have to evaluate $(v_j \circ F_{K})
+(\hat x_i)$. In general, if $\Gamma$ and $\Omega$ are not aligned, the point
+$F_{K}(\hat x_i)$ is completely arbitrary with respect to $\Omega$, and unless
+we figure out a way to interpolate all basis functions of $V_h(\Omega)$ on an
+arbitrary point on $\Omega$, we cannot compute the integral needed for an entry
+of the matrix $C$.
+
+The process is described by the following algorithm (and by the picture below):
+
+- For a given cell $K$ in $\Gamma$ compute the real point $y_i \coloneqq F_{K}
+(\hat x_i)$, where $x_i$ is one of the quadrature points used for the integral
+on $K \subseteq \Gamma$.
+
+- Find the cell of $\Omega$ in which $y_i$ lies, say it's $T$.
+
+- To evaluate the basis function use the inverse of the mapping $G_T$ that
+transforms the reference element $\hat T$ into the element $T$: $v_j(y_i) = \hat
+v_j \circ G^{-1}_{T} (y_i)$.
+
+<p align="center"> <img
+  src="https://www.dealii.org/images/steps/developer/step-60_C_interpolation.png"
+  alt=""> </p>
+
+The three steps above can be computed by calling, in turn:
+
+- GridTools::find_active_cell_around_point, followed by
+
+- Mapping::tranform_real_to_unit_cell
+
+- construct a custom Quadrature formula, containing the point in the reference
+cell
+
+- construct an FEValues object, with the given quadrature formula, and initialized with the cell obtained in the first step.
```

Comment:
You may want to point out that this process is really what `VectorTools::point_value()` does when evaluating a finite element field (not just a single shape function) at an arbitrary point.